files/ko/web/api/clipboard_api/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Clipboard API
+slug: Web/API/Clipboard_API
+tags:
+  - API
+  - Async Clipboard API
+  - Clipboard
+  - Clipboard API
+  - Clipboard Event API
+  - ClipboardEvent
+  - ClipboardItem
+  - Cut
+  - Landing
+  - Reference
+  - copy
+  - paste
+  - 클립보드
+  - 복사
+  - 붙여넣기
+translation_of: Web/API/Clipboard_API
+---
+{{DefaultAPISidebar("Clipboard API")}}
+
+**Clipboard API**는 클립보드 명령(잘라내기, 복사, 붙여넣기)에 응답하거나 시스템 클립보드에 비동기적으로 접근하고 쓸 수 있는 기능을 제공합니다.
+
+> **참고:** 이 API는 [웹 워커](/ko/docs/Web/API/Web_Workers_API)에서 사용할 수 **없습니다**. ({{domxref("WorkerNavigator")}}에 노출되지 않음)
+
+Clipboard API는 {{domxref("document.execCommand()")}}를 사용한 클립보드 접근을 대체하기 위해 디자인되었습니다.
+
+## 클립보드 접근
+
+시스템 클립보드에 접근할 땐 `Clipboard` 객체의 인스턴스를 생성하지 않고, 전역 {{domxref("Navigator.clipboard")}}를 사용합니다.
+
+```js
+navigator.clipboard.readText().then(
+  clipText => document.querySelector(".editor").innerText += clipText);
+```
+
+위의 코드 조각은 클립보드에서 텍스트를 가져와서, `editor` 클래스를 가진 요소의 콘텐츠 뒤에 추가합니다. {{domxref("Clipboard.readText", "readText()")}}는 클립보드의 내용이 텍스트가 아니면 빈 문자열을 반환하므로, 이 코드는 안전합니다.
```

Comment:
```suggestion
위의 코드 조각은 클립보드에서 텍스트를 가져와서, `editor` 클래스를 가진 첫 번째 요소의 콘텐츠 뒤에 추가합니다. {{domxref("Clipboard.readText", "readText()")}}(그리고 {{domxref("Clipboard.read", "read()")}}도)는 클립보드의 내용이 텍스트가 아니면 빈 문자열을 반환하므로, 이 코드는 안전합니다.
```

Reply:
This snippet fetches the text from the clipboard and appends it to the first element found with the class editor. Since [readText()](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/readText) (and [read()](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/read), for that matter) returns an empty string if the clipboard isn't text, this code is safe.

Reply:
원문을 반영해 조금 보완해보았습니다!

Reply:
@yechoi42 반영했습니다~